README.md

```diff
@@ -12,58 +12,72 @@ Heroku Connect Toolbelt Plugin
 
 # Commands
 
-    heroku connect:auth                   - Authenticate a connection to Salesforce
-    heroku connect:preauth                - Store a Salesforce authentication for future use
-    heroku connect:export                 - Export a mapping configuration JSON file
-    heroku connect:import                 - Import a mapping configuration JSON file
-    heroku connect:info                   - Display connection information 
-    heroku connect:mapping:state MAPPING  - Returns the state of a mapping
-    heroku connect:setup                  - Configure a new connection
-    heroku connect:state                  - Return the state flag for a single connection
-    heroku connect:pause                  - Pause a connection
-    heroku connect:resume                 - Resume a connection
-    heroku connect:restart                - Restart a connection
+    heroku connect:auth                    - Authenticate a connection to Salesforce
+    heroku connect:export                  - Export a mapping configuration JSON file
+    heroku connect:import FILE             - Import a mapping configuration JSON file
+    heroku connect:info                    - Display connection information 
+    heroku connect:mapping:state MAPPING   - Return the state of a mapping
+    heroku connect:mapping:delete MAPPING  - Delete an existing mapping
+    heroku connect:mapping:reload MAPPING  - Reload a mapping's data from Salesforce
+    heroku connect:pause                   - Pause a connection
+    heroku connect:resume                  - Resume a connection
+    heroku connect:restart                 - Restart a connection
+    heroku connect:setup                   - Configure a new connection
+    heroku connect:state                   - Return the state flag for a single connection
 
 # Examples
 
-Download the mapping configuration
+Download an existing mapping configuration
 
-    heroku connect:export
+    $ heroku connect:export
+    Saved config-file: app-name-resource-name.json
 
 # Tutorial
 
-Create an authorization into your Salesforce org. Note the result token.
+Make sure you have a Heroku app, with a Postgres database attached
 
-    heroku connect:preauth
+## Add the Heroku Connect add-on to your app
 
-    Token: xxx
+    $ heroku addons:create herokuconnect
 
+## Link the new connection (the Heroku Connect add-on instance) to your Heroku user
 
-Make sure you have a Heroku app, with a Postgres database attached
+    $ heroku connect:info
+
+## Now link the connection to the database, specifying the config var and schema name
+
+    $ heroku connect:setup -d DATABASE_URL -s salesforce
+    Configuring connection with
+     { schema_name: 'salesforce', db_key: 'DATABASE_URL' }
```

Comment:
I agree that the JSON is a bit out of place. We want to at least display it because the arguments are currently optional, but we could probably display it better. I think I'd rather leave that for a follow-up PR that cleans up our output in general to be more in line with the style guide that we were recently pointed to.